Replace broken link to Azure Policy pricing

BrianTJackett opened the pull request:
Replace broken link for Azure Policy pricing that appears to have pointed to a temporary Azure website used at Ignite.  Instead point to the official Azure documentation pricing page for Azure Policy.

azurecla commented:
Hi __@lodipalm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

deneha commented:
MSFT employee - cla-not-required

BrianTJackett committed:
Replace broken link to Azure Policy pricing

Replace broken link for Azure Policy pricing that appears to have pointed to a temporary Azure website used at Ignite.  Instead point to the official Azure documentation pricing page for Azure Policy.

GitHubber17 committed:
Update assign-policy-definition.md
